gef.py

```diff
@@ -1480,14 +1485,15 @@ class malloc_chunk_cls(ctypes.Structure):
         ]
         return malloc_chunk_cls
 
-    def __init__(self, addr: int, from_base: bool = False, allow_unaligned: bool = True) -> None:
+    def __init__(self, addr: int, from_base: bool = False, allow_unaligned: bool = True, _type: str = "") -> None:
```

Comment:
Using `_type` isn't probably very clean.

Probably cleaner to make a class that inherits from `GlibcChunk` for those specific cases.